askama/src/filters/mod.rs

```diff
@@ -502,7 +502,7 @@ pub fn center(src: impl ToString, dst_len: usize) -> Result<impl fmt::Display, I
     center(src.to_string(), dst_len)
 }
 
-/// Count the words in that string
+/// Counts the words in that string
```

Comment:
We're already inconsistent on the verb form and this is unrelated to the changes in this commit, so maybe just leave it for now?

(On the whole I think I prefer "Count"/"Center"/"Capitalize" over "Counts"/"Centers"/"Capitalizes".)

Reply:
Reverted this change.